scripts/ha_pushbackup.script

```diff
@@ -52,10 +52,10 @@
    /tool fetch upload=yes src-path=HA_run-after-hastartup.txt dst-path=HA_run-after-hastartup.rsc address=$haAddressOther user=ha password=$haPassword mode=ftp 
 
    /export file=HA_b2s.rsc
-   /system backup save name=HA_b2s.backup password=p
+   /system backup save name=HA_b2s.backup password=$haPassword
    /tool fetch upload=yes src-path=HA_b2s.rsc dst-path=HA_b2s.rsc address=$haAddressOther user=ha password=$haPassword mode=ftp  
    /tool fetch upload=yes src-path=HA_b2s.backup dst-path=HA_b2s.backup address=$haAddressOther user=ha password=$haPassword mode=ftp  
-   /file print file=HA_restore-backup.rsc; /file set [find name="HA_restore-backup.rsc.txt"] contents="/system backup load name=HA_b2s.backup password=p"
+   /file print file=HA_restore-backup.rsc; /file set [find name="HA_restore-backup.rsc.txt"] contents="/system backup load name=HA_b2s.backup password=$haPassword"
```

Comment:
In my case it generated a restart of the slave router.  this happened when I was executing ha_checkchanges

If we do not consider it a bug, it is possible that I was wrong in the implementation

Reply:
Interesting. A restart as in a crash? I'm a bit confused because ha_checkchanges calls this modified ha_pushbackup only if it detects a change between the pairs and if there are changes, it will cause a restart of the slave by design.

Can you help me understand a bit more about the problem you observed? What was the workflow? What hardware and what RouterOS version?

Reply:
It is a CCR routerboard v6.43.7.  When I saw the restart of the slave, I executed the ha_checkchanges manually.  When in the execution it arrived at password = p it made the reboot.  when changing the password it was fixed. 

With password=$haPassword has worked well

I could not see why the router detects a change over and over again

Reply:
Are you saying that you are seeing changes over and over again when ha_checkchanges ($HASyncStandby) is run?

I don't see how changing it from password=p to password=$haPassword would change this behavior. If changes are detected, the slave is going to reboot. If it doesn't reboot, your devices are not in sync.